components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1077,7 +1077,7 @@
 
 <!-- dust emission tuning factor -->
 <dust_emis_fact                                                         >2.76D0</dust_emis_fact>
-<dust_emis_fact          hgrid="ne30np4"   phys="cam5"                  >0.95D0</dust_emis_fact>
+<dust_emis_fact          hgrid="ne30np4"   phys="cam5"                  >1.38D0</dust_emis_fact>
```

Comment:
what is "hgrid" when pg2 is used?  Not sure this will be picked up.

Reply:
It looks like pg2 uses the same hgrid settings, but adds another keyword "npg='2'"